src/box/iproto.cc

```diff
@@ -73,6 +75,23 @@ enum {
 	 ENDPOINT_NAME_MAX = 10
 };
 
+struct iproto_connection;
+
+struct iproto_stream {
```

Comment:
IMO a commit should not add untested code. Can we add a test that would send some stream messages without net.box support? AFAIR there were some tests in box-py, which worked with raw iproto protocol.